### PR TITLE
Upgrade DownloadPackageV1 Task to NodeJS16

### PR DESCRIPTION
**Pull request by LiliaSabitova**
**Task name**: DownloadPackageV1

**Description**: Upgrading DownloadPackageV1 Task to NodeJS16

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by vmapetr — APPROVED** [review]

**Review by ismayilov-ismayil — APPROVED** [review]

**Comment by LiliaSabitova** [comment]
Hi @phil-hodgson and @aasim, could you please take a look at the PR?

File: Tasks/DownloadPackageV1/make.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "rm": [
+        {
+            "items": [
+                "node_modules/azure-pipelines-tasks-packaging-common-v3/node_modules/azure-pipelines-task-lib"
```
Comment (tintse-thxsky-MSFT) [inline Tasks/DownloadPackageV1/make.json:5]: why is it needed to rm azure-pipelines-task-lib?

**Review by tintse-thxsky-MSFT — CHANGES_REQUESTED** [review]
in the task.json I believe you need to update the agent version 2.144.0

File: Tasks/DownloadPackageV1/make.json (hunk 1 above)
Comment (LiliaSabitova) [reply Tasks/DownloadPackageV1/make.json:5]: We update task-lib in the package.json to the new version. But we are not yet updating azure-pipelines-tasks-packaging-common-v3, which has a dependency on previous version of task-lib. And therefore, the task will contain two versions of task-lib. But the problem is that task-lib is written as a singleton, and therefore it initializes only once, with one of the versions, making it impossible to use two versions simultaneously. So, we remove the dependency of old task-lib. In this case we are having only one version of task-lib in the task.

**Review by tintse-thxsky-MSFT — APPROVED** [review]
LGTM

**Review by kirill-ivlev — APPROVED** [review]

**Pull request merged**